fast64_internal/oot/scene/panels.py

```diff
@@ -43,7 +43,7 @@ def draw(self, context):
             self.drawSceneSearchOp(exportBox, settings.option, "Export")
         settings.draw_props(exportBox)
 
-        if context.scene.fast64.oot.hackerFeaturesEnabled:
+        if bpy.context.scene.fast64.oot.feature_set == "hacker_oot":
```

Comment:
same thing about bpy there, it should be removed

Reply:
Done.